test/extended/authorization/scc.go

```diff
@@ -163,4 +163,51 @@ var _ = g.Describe("[sig-auth][Feature:PodSecurity][Feature:SCC]", func() {
 		o.Expect(err).NotTo(o.HaveOccurred())
 		o.Expect(pod.Annotations[securityv1.ValidatedSCCAnnotation]).To(o.Equal("restricted-v2"))
 	})
+
+	g.It("SCC admission fails for incorrect/non-existent required-scc annotation", func() {
+
+		// Define a pod with specific specifications.
```

Comment:
The blank line between `g.It` and its first code is usually not needed.

Reply:
Fixed